### PR TITLE
[CLDR] LPD-32854 Remove CLDR settings in tomcat setenv and add it to CI test environment

### PR DESCRIPTION
**Pull request by julschong**
@vicnate5 @tinatian 
This PR removes `java.locale.providers=CLDR` from tomcat setenv files
This means that included tomcat bundle will no longer have this java option.

Reason to remove: 
- All JDK9 and up default to use CLDR if not specified.
- Since we are not supporting JDK8 anymore, there is no reason to keep this java option

However, CI test environments still have lots of jdk8 tests, this is why the [second commit](https://github.com/liferay-core-infra/liferay-portal/pull/7347/commits/6d9d90e2553c83187d46496e8158897da97d348b) add a step in CI test script to add the CLDR java option.

The main thing to test for with this change is to make sure that `ci:test:environments` tests logs under different environments still have the CLDR settings.
image

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:environments

**Comment by julschong** [comment]
ci:test:environments

**Comment by julschong** [comment]
weblogic CLDR addition was not successful, need to correct

**Comment by julschong** [comment]
ci:test:environments

**Comment by julschong** [comment]
reviewed each env console and made sure that
- JDK8 tests have CLDR
- JDK21 tests do not have CLDR

Good to go

File: build-test.xml [hunk 1]
```diff
@@ -15891,6 +15891,49 @@ mail.send.blacklist=</echo>
 			</then>
 		</if>
 
+		<get-java-jdk-bundle-version test.batch.name="${test.batch.name}" />
+
+		<if>
+			<and>
```
Comment (tinatian) [inline build-test.xml:15897]: @julschong Please remove <and> here

File: build-test.xml [hunk 2]
```diff
@@ -15891,6 +15891,47 @@ mail.send.blacklist=</echo>
 			</then>
 		</if>
 
+		<get-java-jdk-bundle-version test.batch.name="${test.batch.name}" />
+
+		<if>
+			<equals arg1="${java.jdk.bundle.version}" arg2="8" />
```
Comment (julschong) [inline build-test.xml:15897]: <and> removed

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
sf passed, these 2 commits are good to go

**Comment by julschong** [comment]
for unit test's CLDR removal, testing here: https://github.com/julschong/liferay-portal/pull/1327

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/153412

**Pull request closed without merging**